packages/styles/stories/Components/calendar/calendar.stories.js

```diff
@@ -96,6 +97,18 @@ These modifier classes are used to style various states of calendar grid element
 | \`is-hover\` | Style hovered calendar elements
 | \`is-disabled\` | Style disabled calendar elements
 
+
+### Legend
+These modifier classes are used to style calendar grid elements with colors.\n
+| Modifier class | Use to... |
+| -------------:| :-------------- |
+| \`fd-calendar__row--legend\` | Create a row space for legend items 
+| \`fd-calendar--legend__item\` | Create a legend container 
+| \` fd-calendar--legend__item--circle\` | Makes the legend item representation a circle
+| \` fd-calendar--legend__item--square\` | Makes the legend item representation a square
+| \` fd-calendar--legend__item--text\` | Adds a descriptive title to the legend item
+| \` fd-calendar--legend__item--text-description\` | Adds any additional information to the legend item
```

Comment:
These class names are not following the BEM rules.
`fd-calendar` is the block element, because the entire element can be "extracted" from a project and used in another project, aka it's an independent unit. The elements inside this blocks have the double underscore in their class names. 
For example: `fd-calendar__row`, `fd-calendar__item`, `fd-calendar__date`, etc.
When you look at these classes you can immediately tell that it's a row/item/date, etc. that belong to the calendar component (block). 
The double dash notation is a modifier. If you have an item, or date, that are a variation of the common one, you use a modifier class to add specifics for it. For example, if the weekend dates and the work days have different background, then you do something like this:
`fd-calendar__date fd-calendar__date--weekend` or `fd-calendar__date fd-calendar__date--work`, `fd-calendar__date fd-calendar__date--vacation`.
When you read the class `fd-calendar__date--vacation` it basically says: "it's a vacation variation of date element that belongs to calendar". 
If you have to read this class `fd-calendar--legend__item--circle`, you can't tell what is an element and what is a modifier.

Reply:
> here's more on BEM: 
https://getbem.com/introduction/
https://en.bem.info/methodology/css/
https://css-tricks.com/bem-101/